EIPS/eip-2981.md

```diff
@@ -139,15 +149,15 @@ Additionally, we return a specific `_royaltyAmount` so there is no dispute with
 
 ### Unused data parameters in function signature
 
-This EIP mandates the inclusion of the `_data` parameter and the `_royaltyPaymentData` return value, and also mandates that these are unused. The purpose of this is to have a seamless path to build on this ERC by further EIP standards that specify payment notifications, fee splitting, and other advanced use cases.
+This EIP mandates the inclusion of the `_data` parameter and the `_royaltyPaymentData` return value, and also mandates that these are unused. The purpose of this is to have a seamless path to build on this ERC by further EIP standards that specify payment notifications, fee splitting, and other advanced use cases. Using them is considered undefined behavior.
 
 ## Backwards Compatibility
 
 This standard is compatible with current ERC-721 and ERC-1155 standards.
 
 ## Universal Royalty Payments
```

Comment:
This section should be made a subsection of a valid EIP section (see EIP-1 for a list).

Reply:
This change seems out of step with the current discussion, which is leaning toward removal of these unused param/return values in favor of letting future EIPs simply overload or add methods as required.

Reply:
I'm not sure if I follow. I am requesting that you move the section "Universal Royalty Payments" from the `##` level to at least `###` under a valid EIP section. You can find a list of valid sections in EIP-1.

Reply:
@lightclient No worries, I see what's happened. This PR from the other day was stuck, so I'm seeing all those changes, not changes you made.